azure-pipelines.yml

```diff
@@ -1,19 +1,42 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
-
 trigger:
 - main
+- master
+- staging/*
+- dev/*
+
+pr:
+- main
+- master
 
-pool:
-  vmImage: 'ubuntu-latest'
+variables:
+  isMain: $[eq(variables['Build.SourceBranch'], 'refs/heads/master')]
```

Comment:
isMain doesn't look used;
runCondition seems always true; will it be used?
targetBranch/TARGET_BRANCH doesn't seem used ; will it be used?

Reply:
done. removed the unused variable